Demos/MaterialCreator/MaterialCreator.csproj

```diff
@@ -11,8 +11,8 @@
     <AssemblyName>MaterialCreator</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16225.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16225.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16257.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.16257.0</TargetPlatformMinVersion>
```

Comment:
It just got released an hour ago, but let me double check that it loads. I'll sort back asap with results.